src/webapp/pages/analysis/steps/3-general-practitioners/GeneralPractitionersStep.tsx

```diff
@@ -56,9 +58,33 @@ export const GeneralPractitionersStep: React.FC<PageStepProps> = React.memo(prop
                 />
             </StepAnalysis>
         </>
+    ) : (
+        <>
+            <AnalysisHeader>
+                <StyledTypography variant="h2">{title}</StyledTypography>
+            </AnalysisHeader>
+            <EmptyState
+                message={i18n.t("The selected step is not available for the selected module")}
+                variant={"neutral"}
```

Comment:
No need to change, but you can omit the curly brackets since this is a plain string

```tsx
variant="neutral"
```